staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2444,6 +2444,11 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// This MUST be set only for init containers.
```

Comment:
This is a confusing statement.

"This may only be set for initContainers" is more correct.  In particular, an initContainer is allowed to NOT specify this, right?

Reply:
I am still looking for some explanation of what this does, and what happens if it is not set. 

It's very unusual to not have a default value...

Reply:
> In particular, an initContainer is allowed to NOT specify this, right?

yes, that is right.